src/vendor/webauthn.js

```diff
@@ -0,0 +1,185 @@
+"use strict";
+define(function () {
+
+  var webauthn = (function () {
+
+    const webauthnDB = (function () {
```

Comment:
When you tested it earlier, why didn't it break? Is it a silent error (i.e. something that just warns to the console but doesn't actually affect the rest of the script running)?